csharp/Platform.Collections/ICollectionExtensions.cs

```diff
@@ -16,17 +16,37 @@ public static class ICollectionExtensions
         /// <para>Checking collection for empty.</para>
         /// <para>Проверяет коллекцию на пустоту.</para>
         /// </summary>
+        /// <typeparam name="T">
+        /// <para>Collection element data type.</para>
+        /// <para>Тип данных элемента коллекций.</para>
+        /// </typeparam>
         /// <param name="collection">
         /// <para>Method takes an elements collection of <see cref="ICollection<T>"/> type.</para>
-        /// <para>Метода принимает колекцию элементов <see cref="ICollection<T>"/> типа.</para>
+        /// <para>Метод принимает колекцию элементов <see cref="ICollection<T>"/> типа.</para>
         /// </param>
         /// <returns>
-        /// <para>Returns a <see cref="bool"/> type variable equal to False if the collection is empty else returns true.</para>
+        /// <para>Returns a <see cref="bool"/> type variable equal to false if the collection is empty else returns true.</para>
```

Comment:
You can say just **The `<see cref="bool"/>` type variable equal to false if the collection is empty else returns true.**